sox/file_info.py

```diff
@@ -21,17 +21,48 @@ def bitrate(input_filepath):
 
     Returns
     -------
-    bitrate : int
-        number of bits per sample
-        returns 0 if not applicable
+    bitdepth : int or None
+        Number of bits per sample.
+        Returns None if not applicable.
     '''
     validate_input_file(input_filepath)
     output = soxi(input_filepath, 'b')
     if output == '0':
-        logger.warning("Bitrate unavailable for %s", input_filepath)
+        logger.warning("Bit depth unavailable for %s", input_filepath)
+        return None
     return int(output)
 
 
+def bitrate(input_filepath):
+    '''
+    Bit rate averaged over the whole file.
+    Expressed in bytes per second (bps), or None if not applicable.
+
+    Parameters
+    ----------
+    input_filepath : str
+        Path to audio file.
+
+    Returns
+    -------
+    bitrate : float or None
+        Bit rate, expressed in bytes per second.
+        Returns None if not applicable.
+    '''
+    validate_input_file(input_filepath)
+    output = soxi(input_filepath, 'B')
+    # The characters below stand for kilo, Mega, Giga, etc.
+    greek_prefixes = '\0kMGTPEZY'
+    if output == "0":
+        logger.warning("Bit rate unavailable for %s", input_filepath)
+        return None
+    elif output[-1] in greek_prefixes:
+        multiplier = 1000**(greek_prefixes.index(output[-1]))
```

Comment:
Use a float here so we're sure `multiplier` is also a float. Or should everything be cast to the nearest integer?
`multiplier = 1000.0**(greek_prefixes.index(output[-1]))`

Reply:
fixed in 30d4d91